tools/submission/submission_checker.py

```diff
@@ -2381,6 +2413,39 @@ def check_measurement_dir(
             if not os.path.exists(os.path.dirname(code_dir)):
                 log.error("%s is missing code_dir %s", fname, code_dir)
                 is_valid = False
+  
+        # Check equal issue mode        
```

Comment:
I don't think we should do this check in mlperf.conf file as it can be overridden in user.conf file. Moreover users might mistakenly put wrong/stale mlperf.conf file if they are using the default mlperf.conf file. So we should check this in mlperf_log_detail file only.

Reply:
@arjunsuresh As I pointed in the LLM chat. The issue is that we are not logging the equal issue mode in the `mlperf_log_detail`. This PR also makes it so that this is logged, but we can't check if previous submissions ran with the equal issue mode based in the `mlperf_log_detail`. 

We can check the `mlperf_log_detail`, `mlperf.conf` and `user.conf`, and the submission checker passes if one of them has the equal issue